app/src/main/res/drawable/top_sites_background.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:shape="rectangle">
+    <corners android:radius="4dp" />
```

Comment:
Looking at figma https://www.figma.com/file/lfgj15zbNkL9Pp4hdZGJgp/Proton-Mobile-UI?node-id=809%3A1. I see a radius of 8dp. I feel like we should update this here as well. I don't know how it will look with 8dp radius so we might need another UX review with this change to be on the safe side.

<img width="520" alt="Screen Shot 2021-04-19 at 12 44 14 PM" src="https://user-images.githubusercontent.com/1190888/115272974-fbbbcb00-a10c-11eb-8261-63b7568a9461.png">